### PR TITLE
Newsletters: add 298 (2024-04-17)

### PR DESCRIPTION
**Pull request by harding**
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Client/Services section @bitschmidty 

~~I'll check later in the week/weekend to look again for possible news stories or PR merges, but I'm guessing a lot of people are busy with the conference, so it might be a slow news week.~~

### CODE REVIEW

**Comment by bitschmidty** [comment]
Might consider: https://delvingbitcoin.org/t/research-into-the-effects-of-a-cluster-size-limited-mempool-in-2023/794

**Review by murchandamus — COMMENTED** [review]
LGTM so far. I have no improvement suggestions at this time.

Edit:
Looks like 27.0 has been tagged: https://github.com/bitcoin/bitcoin/releases/tag/v27.0

**Comment by harding** [comment]
Added lede, releases/RCs, and topic entries.  Added FIXME for @bitschmidty to remove the Bitcoin Core 27 release announcement if the binaries haven't been uploaded to BitcoinCore.org by Wednesday morning.  Thanks @bitschmidty and @murchandamus for the reviews!

File: _posts/en/newsletters/2024-04-17-newsletter.md [hunk 1]
```diff
@@ -92,10 +96,37 @@ FIXME:bitschmidty
...
+  default (see [Newsletter #288][news288 v2 p2p]), allows the use of
+  opt-in topologically restricted until confirmation ([TRUC][topic v3
+  transaction relay]) transaction policy (also called _v3 transaction
+  relay_) on test networks (see [Newsletter #289][news289 truc]), and
+  adds a new [coin selection][topic coin selection] strategy to be used
+  during high feerates (see [Newsletter #290][news290 coingrinder]).
+  For a complete list of major changes, please see the [release
+  notes][bcc27 rn].
+
+- [BTCPay Server 1.13.1][] is the is the latest release of this
```
Comment (murchandamus) [inline _posts/en/newsletters/2024-04-17-newsletter.md:116]:
```suggestion
- [BTCPay Server 1.13.1][] is the latest release of this
```

File: _posts/en/newsletters/2024-04-17-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+summarizing notable changes to popular Bitcoin infrastructure software.
+
+## News
+
+- **What would have happened if cluster mempool had been deployed a year ago?**
+  Suhas Daftuar [posted][daftuar cluster] to Delving Bitcoin that he
+  recorded every transaction his node received in 2023 and has now run
+  them through a development version of Bitcoin Core with [cluster
+  mempool][topic cluster mempool] enabled to quantify differences
+  between the two systems.  Some of his findings include:
```
Comment (bitschmidty) [inline _posts/en/newsletters/2024-04-17-newsletter.md:23]: It may not be clear what the two systems are here

**Comment by murchandamus** [comment]
27.0 released: https://github.com/bitcoin/bitcoin/releases/tag/v27.0

**Comment by harding** [comment]
Pushed edits for @murchandamus and @bitschmidty review comments (thanks!) and reviewed client/services section (looks great, thanks @bitschmidty !).

**Review by bitschmidty — APPROVED** [review]
Updates look good, took the suggestion from @LarryRuane, squashed

**Pull request merged**